tests/testConverterGeometries.py

```diff
@@ -0,0 +1,171 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Newton Developers
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import usdex.core
+from pxr import Gf, Usd, UsdGeom
+
+from tests.util.ConverterTestCase import ConverterTestCase
+from urdf_usd_converter._impl.convert import Converter
+
+
+class TestConverterGeometries(ConverterTestCase):
+    def setUp(self):
+        super().setUp()
+        self.tolerance = 1e-6
+
+    def test_geometries(self):
+        input_path = "tests/data/simple-primitives.urdf"
+        output_dir = self.tmpDir()
+
+        converter = Converter()
+        asset_path = converter.convert(input_path, output_dir)
+        self.assertIsNotNone(asset_path)
+        self.assertTrue(pathlib.Path(asset_path.path).exists())
+
+        stage: Usd.Stage = Usd.Stage.Open(asset_path.path)
+        self.assertIsValidUsd(stage)
+
+        default_prim = stage.GetDefaultPrim()
+        geometry_scope_prim = stage.GetPrimAtPath(default_prim.GetPath().AppendChild("Geometry"))
+        self.assertIsNotNone(geometry_scope_prim)
```

Comment:
This would never be None, you probably want to be using `self.assertTrue(prim.IsValid())`